packages/web-app-admin-settings/tests/unit/components/AppTemplate.spec.ts

```diff
@@ -60,7 +60,7 @@ describe('AppTemplate', () => {
         const { wrapper } = getWrapper({
           propsData: { breadcrumbs: [{ text: 'Administration Settings' }, { text: 'Users' }] }
         })
-        expect(wrapper.find(stubSelectors.ocBreadcrumb).props().items).toEqual([
+        expect(wrapper.findComponent<any>(stubSelectors.ocBreadcrumb).props().items).toEqual([
```

Comment:
Sure. `find` just finds a `DOMElement`, `findComponent` finds a `Component` - suprising, eh? :-D

In this case we need to find the component because otherwise `.props()` is not available.

Reply:
😁 🙇